### PR TITLE
read from files and build trie

### PR DESCRIPTION
**Pull request by davidcoh464**

### CODE REVIEW

File: read_to_trie.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+from trie import Trie
+import os
+import re
+
+pattern = r'[^a-zA-Z0-9\s]'
+
+
+def read_files(trie: Trie, dir_path: str, file_index: int = 0) -> None:
```
Comment (AsherMentzer) [inline read_to_trie.py]: You need to insert all the words to the data structure

File: Archive/0130903086.txt
Comment (otapiero) [inline Archive/0130903086.txt:1]: do not load text files to git

File: read_to_trie.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    Returns:
+        None
+    """
+    for file_name in os.listdir(dir_path):
+        file_path = os.path.join(dir_path, file_name)
+        if os.path.isfile(file_path):
+            if file_name.endswith(".txt"):
+                with open(file_path, 'r', encoding='utf-8') as file:
+                    for line_number, line in enumerate(file):
+                        clean_line = re.sub(pattern, '', line).lower().strip()
```
Comment (otapiero) [inline read_to_trie.py]: separate to differents functions

**Review by otapiero — CHANGES_REQUESTED** [review]
please fix the code before

File: read_to_trie.py (hunk 2 above)
Comment (davidcoh464) [reply read_to_trie.py]: done

File: Archive/0130903086.txt
Comment (davidcoh464) [reply Archive/0130903086.txt:1]: done

File: read_to_trie.py (hunk 1 above)
Comment (davidcoh464) [reply read_to_trie.py]: done

**Pull request merged**